api/petitions.api.js

```diff
@@ -2,8 +2,15 @@ const express = require("express");
 const router = express.Router();
 
 const petitionsController = require("../controllers/petitions.controller");
+const { loginRequired } = require("../middlewares/auth.middleware");
+
+/**
+ * @route : /fund
+ * @description : Create Fund Request petition or Fund Donate to a FR petition
+ * @access: Login required
+ */
```

Comment:
I'm of the opinion your `petitionsController.createWithFund` is explicit so we don't need a comment. You don't have to remove it to merge, I'm just sharing my opinion.